indigo/pipelines/hier.py

```diff
@@ -30,7 +30,8 @@ class IdentifySections(Stage):
 
     def __call__(self, context):
         for p in context.html.xpath('./p'):
-            text = ''.join(p.xpath('.//text()'))
+            # lstrip text in case there's inline whitespace at the start
+            text = ''.join(p.xpath('.//text()')).lstrip()
```

Comment:
Can you tweak one of the existing test fixture files so it covers this?

Reply:
ie. it would have failed before but now passes. So that we don't regress later accidentally.

Reply:
good idea, thanks!